load-map.js

```diff
@@ -1,3 +1,5 @@
+var echartslayer;
+
 var map = new mapboxgl.Map(

```

Comment:
After playing around with my location map for a bit, I realized that we should maybe display city names or something so it's easy to have bearings on where we are.  If this is possible, we can make this a future enhancement.

No resolution needed.

Reply:
Okay, I'll dig into that.

Reply:
+1